tox.ini

```diff
@@ -28,20 +27,17 @@ max-doc-length = 120
 ; D413 = Missing blank line after last section (numpy style)
 ; D414 = Section has no content (numpy style)
 ignore = D101,D200,D203,D212,D215,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414
-match-dir = (?!migrations)
 
 [pytest]
 DJANGO_SETTINGS_MODULE = test_utils.test_settings
-addopts = --cov openedx_events --cov tests --cov-report term-missing --cov-report xml
+addopts = --cov openedx_events --cov-report term-missing --cov-report xml
```

Comment:
since we are removing `tests` from coverage, should we also remove the dir as it contains nothing but placeholders?
https://github.com/openedx/openedx-events/tree/main/tests

Reply:
Yes! Done. Thank you